youtube.py

```diff
@@ -4,7 +4,6 @@
 import requests
 from lib.config import Config
 
-
```

Comment:
I think the PEP8 standard _might_ be 2 lines between imports and the class def'n but no big deal (minor semantics for a small project)